### ISSUE TITLE
Symbol list uses 100% CPU

### ISSUE THREAD

**Issue by pedro-ricardo** [krvajal/vscode-fortran-support#71]
The code below freezes the extension when the **Symbol List** (Ctrl+Shift+O) command is used. The extension keeps running in the background with 100% CPU even after VScode is closed.
```
module symbol_test
    
    implicit none
    
contains
    
    subroutine Much_dummy_arguments(headlev, rhs, rho, rhof, mu, u, u1, v, v1, w, w1, sx, sy, sz, &
        kindv, crlx, rlx_down, rlx_up, shift, t, dt, alfa, theta, tol_vel, ap_simple)
        implicit none
        
    end subroutine Much_dummy_arguments
    
end module symbol_test
```
The screenshot below illustrates what is happening:
![symb_err](https://user-images.githubusercontent.com/15893711/41126063-ab83387c-6a7c-11e8-9561-50303b0debcd.png)
After pressing Ctrl+Shift+O it is not possible to type anything, the editor freezes.
If you kill the code process that is using 100% CPU it is possible to type in editor once again, but the symbol list shows `No symbol information for the file` and the following window pops up:
![err](https://user-images.githubusercontent.com/15893711/41126390-b8a66ef6-6a7d-11e8-8c70-86c267318aa0.png)

**Comment by pedro-ricardo** [comment]
Error does not happen if user setting`"fortran.symbols": [ ]` is blank.

**Comment by pedro-ricardo** [comment]
Most likelly the error was introduced by commit #64

**Comment by pedro-ricardo** [comment]
Fixed in PR #72
